src/NzbDrone.Core/MediaCover/MediaCoverResizer.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using ImageResizer;
+using NzbDrone.Common.EnsureThat;
+
+namespace NzbDrone.Core.MediaCover
+{
+    public interface IMediaCoverResizer
+    {
+        void ResizeMediaCover(string sourceFileName, string resizedFileName, int? width, int? height);
+    }
+
+    public class MediaCoverResizer : IMediaCoverResizer
+    {
+        public void ResizeMediaCover(string sourceFileName, string resizedFileName, int? width, int? height)
```

Comment:
what if they are both null? I think we should make this width only and that would be required.